build.sh

```diff
@@ -2,6 +2,11 @@
 
 set -euox pipefail
 
+# See https://github.com/centos-workstation/achillobator/issues/3
+mkdir -m 0700 -p /var/roothome
```

Comment:
Also forgot to mention in the commit message but `-m 0700` is important here too